util/GetName.ts

```diff
@@ -10,3 +12,16 @@ export function getName(firstName?: string, lastName?: string): string | undefin
       return undefined;
   }
 }
+
+export function getNameFromUserDetails(userDetails: UserDetails): string {
```

Comment:
could we just reuse above function and in case return of undefined return userDetails userName ?